content/en/docs/reference/glossary/finalizer.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Finalizer
+id: finalizer
+date: 2021-07-07
+full_link: /docs/concepts/overview/working-with-objects/finalizers/
+short_description: >
+  A field in the Kubernetes API that specifies what actions to take
```

Comment:
This short description does not seem clear -- especially the start of the sentence,
"A field in the Kubernetes API". There are a lot of fields ....
Compare to:
In Kubernetes, finalizers are keys that allow you to specify pre-delete
operations for resources.

Reply:
Would "A Kubernetes API resource that specifies.." work better? It was originally "A resource key that specifies" but we changed it in [this review comment](https://github.com/kubernetes/website/pull/28870#discussion_r666544260)

Reply:
Finalizers aren't fields nor are they resources with URIs.
To me they are _identifiers_, namespaced similar to the keys for labels and annotations. AIUI: Kubernetes itself pays no heed to naming conventions for finalizers. The API server simply waits for the finalizer list to become empty as each defined cleanup mechanism completes.

Reply:
For more help on this, maybe ask in [#wg-api-expression](https://kubernetes.slack.com/messages/wg-api-expression) on Kubernetes Slack.

Reply:
[Asked!](https://kubernetes.slack.com/archives/C0123CNN8F3/p1626183692078400)

Reply:
I changed the definition a bit to make it clearer (hopefully). @sftim PTAL. If it's OK, I have no other changes for this PR.